examples/Makefile.am

```diff
@@ -14,7 +14,7 @@ noinst_PROGRAMS = \
 
 
 AM_CFLAGS = $(WERROR_CFLAGS) $(WARN_CFLAGS) -DMALLOC_RETURNS_NONNULL
-AM_CPPFLAGS = -I$(top_srcdir)/include/wget \
```

Comment:
The examples do not need gnulib (`lib/`). If there is any issue with Windows/MSVC, please use `#ifdef`.